packages/plugins/io-center/src/ui/components/lp-list/lp-element.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+    import { CheckCircle2, Link } from "lucide-svelte";
+    import type { LpElement } from "./types.lp-list";
+    import { addLpElementToCanvas } from "@/headless/stores/canvas-operations.svelte";
+    import { canvasStore } from "../canvas/canvas-store.svelte";
+
+    type Props = {
+        lpElement: LpElement;
+        searchTerm: string;
+    };
+
+    let { lpElement, searchTerm }: Props = $props();
+
+    const { name, isLinked } = lpElement;
+
+    let isSearched = $derived(
+        searchTerm !== "" &&
+            lpElement.name.toLowerCase().includes(searchTerm.toLowerCase()),
+    );
+
+    let isSelected = $derived(
+        canvasStore.logicalPhysicals.some((item) => item.id === lpElement.id),
+    );
+
+    const baseClass =
+        "font-mono cursor-pointer hover:no-underline rounded-md hover:bg-gray-100 transition-colors duration-300";
+
+    function getSelectedClass() {
+        return isSelected ? "bg-beige hover:bg-beige" : "";
+    }
+
+    function getSearchedClass() {
+        return isSearched ? "bg-gray-200 hover:bg-gray-200" : "";
+    }
+</script>
+
+<button
+    class={`flex items-center gap-2 text-lg p-1 w-full mb-1 ${baseClass} ${getSelectedClass()} ${getSearchedClass()}`}
```

Comment:
why not but `flex items-center gap-2 text-lg p-1 w-full mb-1` also into the `baseClass`?